vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/implementation/VLogReasoner.java

```diff
@@ -129,6 +130,15 @@ public boolean equals(Object obj) {
 			return predicate.equals(other.predicate);
 		}
 
+		public String getSyntacticRepresentation() {

```

Comment:
we don't want a syntactic representation for a reasoner. Btw, how does this even compile? There is no getString(Reasoner) in the Serializer.

Reply:
because it has a local class that implements DataSourceDeclaration and I don't know what to do about it because DataSourceDeclaration extends entity